src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,19 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard';
+import { Movies } from '../../types/Movies';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  props: Movies[];
+};
+
+export const MoviesList: React.FC<Props> = ({ props }) => (
+  <div className="movies">
+    {props.map((movie) => (
+      <MovieCard
+        key={movie.imdbId}
+        {...movie}
```

Comment:
Get rid of spread operator.
https://mateacademy.slack.com/archives/C02CSGP1KD1/p1636094929188700